datastream/datastream.py

```diff
@@ -377,14 +378,33 @@ def load_state_dict(self, state_dict):
 
 
 class Datastream(BaseModel):
+    '''
+    ``Datastream`` combines a ``Dataset`` and a sampler into a stream of
+    examples.
+
+        >>> from datastream import Dataset, Datastream
+        >>> data_loader = (
+        ...     Datastream(
+        ...         Dataset.from_subscriptable([1, 2, 3])
+        ...     )
+        ...     .data_loader(batch_size=16, n_batches_per_epoch=100)
+        ... )
+        >>> len(next(iter(data_loader)))
```

Comment:
smaller example and actually show the result of next

Reply:
Cannot show the result of next. It is stochastic and this is a doctest

Have any suggestions for a smaller example?

Reply:
Same example but I removed two newlines. I don't think I can make it smaller.

```
        >>> from datastream import Dataset, Datastream
        >>> data_loader = (
        ...     Datastream(Dataset.from_subscriptable([1, 2, 3]))
        ...     .data_loader(batch_size=16, n_batches_per_epoch=100)
        ... )
        >>> len(next(iter(data_loader)))
        16
```